src/action/macos/encrypt_apfs_volume.rs

```diff
@@ -171,13 +173,32 @@ impl Action for EncryptApfsVolume {
 
         let disk_str = &self.disk.to_str().expect("Could not turn disk into string"); /* Should not reasonably ever fail */
 
-        execute_command(
-            Command::new("/usr/sbin/diskutil")
-                .arg("mount")
-                .arg(&self.name),
-        )
-        .await
-        .map_err(Self::error)?;
+        let mut retry_tokens: usize = 60;
```

Comment:
Why 60?

Reply:
I figure 30s isn't so bad ... I dunno :). Alt?

Reply:
Well, all the other places we retry we've been doing 5s (10 times) so I probably would have started there

Reply:
```suggestion
        let mut retry_tokens: usize = 10;
```

Reply:
Done!